guides/araujo/FTP Writeup.md

```diff
@@ -0,0 +1,92 @@
+<p>Choose “Add/Remove Windows Components”.  Here, you can add or remove all the different components that Windows Server 2003 offer.  The components that we are looking for are within the “Application Server” component.  Once checked, click on “Details” to make sure that “Internet Information Services (IIS)” is checked.  click on the details for IIS and make sure that the FTP service is selected.</p>
+<p><img src="/pics/Capture.PNG" alt=""></p>
+<p><img src="/pics/Capture1.PNG" alt=""></p>
+<p><img src="/pics/Capture2.PNG" alt=""></p>
+<p>Click OK and continue through the rest of the installation process and let it finish.  Once that is done, click on the start button and mouse over “Administrative Tools” then click on “Internet Information Services (IIS) Manger”</p>
+<p>Here is the main interface in which you’ll be configuring and operating your FTP server.  This also allows you to configure and operate a web server, but for the purposes of this guide, we will stick to FTP exclusively.</p>
+<p><img src="/pics/Capture4.PNG" alt=""></p>
+<p>Click on your computer on the left panel and a folder names “FTP Sites” should appear.  If you open the folder, you’ll notice that there is already a FTP server running named “Default FTP Site”.  Since we are going to make our own server, you’ll want to stop it and delete it by right-clicking on the site.</p>
+<p>Right-click on the “FTP Sites” folder and mouse-over “New”, then click on “FTP Site”.  Add a description for the FTP site that you find appropriate then add the same IP address (172.20.241.9) that you set for the server in the following box and make sure that the site is running on port 21.</p>
+<p>Depending on the situation, you’ll want to select the appropriate user isolation policy for this server.  As an example, the SECCDC competition involves an Active Directory server, so in that situation, you’d want to Isolate users using Active Directory, but for the purposes of this tutorial, select “Do not isolate users”.</p>
+<p>For security purposes, DO NOT place your home directory for your FTP server in the default “inetpub/ftproot” folder.  Make sure to create a separate directory just for the FTP server and select that as your root.  Finally, set your initial permissions to read and write for now, you’ll be able to change them whenever you’d like.</p>
+<p>At this point, you should have a FTP site that is up and running.  Right click on the site and select “Properties”, then click on the “Security Accounts” tab and change the anonymous username and password to test out the service.</p>
+<p><img src="/pics/Capture3.PNG" alt=""></p>
+<p>Use the Windows 10 client machine to test out the service by going on the web browser and typing in “<a href="ftp://172.20.241.9">ftp://172.20.241.9</a>”.  You should be prompted with a login and by using the anonymous login credentials you previously set, login to the server and place a test file to see if the service works properly.</p>
```

Comment:
Again code comments would be awesome here. Though you should also test the connection with cmd and use the `ftp` command.